docs/stdlib/enum.rst

```diff
@@ -11,24 +11,24 @@ Enums
 
     :index: enum
 
-    An enumerated type is a data type consisting of an ordered list of values.
+    Represents an enumeration of values within an ordered list.
 
-    An enumerated type can be declared in a schema declaration using
-    the following :ref:`syntax <ref_datamodel_enums>`:
+    An enumerator type can be declared in a schema definition by using
+    the :ref:`following syntax <ref_datamodel_enums>`:
```

Comment:
The label here doesn't reflect what we're linking very well in my opinion. Since we already follow this with an example, I'd lean toward just removing the link here. I don't think it adds much value over the inline example.
```suggestion
    the following syntax:
```